utils/utils.py

```diff
@@ -427,7 +424,7 @@ def try2read_csv(path_file, in_case_of_path, msg):
         Path(path_file).resolve(strict=True)
     except FileNotFoundError:
         if in_case_of_path:
-            path_file = os.path.join(in_case_of_path, os.path.basename(path_file))
+            path_file = str(Path(in_case_of_path) / (path_file.split('./')[-1]))
             try:
```

Comment:
I'll get back to this in_case_of_path story in a future PR. Would like to suggest a different, lighter approach with hydra.utils.get_original_cwd()